data/core/docview.lua

```diff
@@ -563,7 +563,7 @@ function DocView:draw_overlay()
     local T = config.blink_period
     for _, line1, col1, line2, col2 in self.doc:get_selections() do
       if line1 >= minline and line1 <= maxline
-      and system.window_has_focus(core.window) then
+      and core.windows[1]:has_focus() then
```

Comment:
Ideally we'll want to set this to look at the docview's rootview and tied window, but this is good enough for now.